srcpkgs/widelands/template

```diff
@@ -1,21 +1,23 @@
 # Template file for 'widelands'
 pkgname=widelands
-version=21
-revision=5
-wrksrc="${pkgname}-build${version}"
+# Widelands 1.0 release supercedes beta builds
+reverts="21_5"
+version=1.0
+revision=1
+wrksrc="${pkgname}-${version}"
 build_style=cmake
 configure_args="-DOPENGL_INCLUDE_DIR=${XBPS_CROSS_BASE}/usr/include
  -DOPTION_BUILD_WEBSITE_TOOLS=OFF -DWL_INSTALL_DATADIR=/usr/share/widelands"
 hostmakedepends="python3 pkg-config gettext"
 makedepends="boost-devel icu-devel minizip-devel gettext-devel
- glu-devel glew-devel SDL2-devel SDL2_gfx-devel SDL2_image-devel
+ glu-devel glew-devel libcurl-devel SDL2-devel SDL2_gfx-devel SDL2_image-devel
  SDL2_net-devel SDL2_mixer-devel SDL2_ttf-devel"
 short_desc="Real-time strategy game"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="travankor <travankor@tuta.io>"
 license="GPL-2.0-or-later"
 homepage="http://www.widelands.org/"
-distfiles="https://github.com/widelands/widelands/archive/build${version}.tar.gz>${pkgname}-${version}.tar.gz"
-checksum=601e0e4c6f91b3fb0ece2cd1b83ecfb02344a1b9194fbb70ef3f70e06994e357
+distfiles="https://github.com/widelands/widelands/archive/v${version}.tar.gz>${pkgname}-${version}.tar.gz"
```

Comment:
do we still need `>${pkgname}-${version}`?

Reply:
`$wkrsrc` would be `v${version}` otherwise